### ISSUE TITLE
Builderator gemspec includes itself

### ISSUE THREAD

**Issue by fmitchell-r7** [rapid7/builderator#79]
I'm running the following commands to build and install Builderator locally.

``` bash
gem build builderator.gemspec
gem install builderator-1.1.5.gem
```

The gem builds, and installs, but validation fails with an error.

``` text
The gemspec at builderator.gemspec is not valid. Please fix this gemspec. (Gem::InvalidSpecificationException)
The validation error was 'builderator-1.1.5 contains itself (builderator-1.1.5.gem), check your files list'
```

We should probably [exclude Builderator from its gemspec](https://github.com/rapid7/builderator/blob/1.1.5/builderator.gemspec#L16).

**Comment by erran-r7** [comment]
`Dir['**/*']` hehe. :trollface:

**Comment by erran-r7** [comment]
@fmitchell-r7 We could use [bundler's default files glob](https://github.com/bundler/bundler/blob/667d4e44/lib/bundler/templates/newgem/newgem.gemspec.tt#L28-L30):

``` ruby
spec.files = `git ls-files -z`.split("\x0").reject do |f|
  f.match(%r{^(test|spec|features)/})
end
```

**Comment by athompson-r7** [comment]
Fixed in #87
